src/main/java/cinema/model/Role.java

```diff
@@ -0,0 +1,55 @@
+package cinema.model;
+
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "roles")
+public class Role {
+    @Id
+    @Column(unique = true)
+    @Enumerated(value = EnumType.STRING)
+    private RoleName roleName;
```

Comment:
user Long id for PK 
with your current impl it'll be more complicated if your want to update RoleName, etc